PowerEditor/src/NppCommands.cpp

```diff
@@ -3745,8 +3745,9 @@ void Notepad_plus::command(int id)
 		case IDM_VIEW_CURLINE_HILITING:
 		{
 			COLORREF colour = (NppParameters::getInstance()).getCurLineHilitingColour();
-			_mainEditView.setCurrentLineHiLiting(!_pEditView->isCurrentLineHiLiting(), colour);
-			_subEditView.setCurrentLineHiLiting(!_pEditView->isCurrentLineHiLiting(), colour);
+			bool hilite{ NppParameters::getInstance().getSVP()._currentLineHilitingShow };

```

Comment:
```cpp
COLORREF colour = (NppParameters::getInstance()).getCurLineHilitingColour();
...
bool hilite{ NppParameters::getInstance().getSVP()._currentLineHilitingShow };
```
could be 
```cpp
NppParameters& nppParams = NppParameters::getInstance();
COLORREF colour = nppParams.getCurLineHilitingColour();
...
bool hilite{ nppParams.getSVP()._currentLineHilitingShow };
```
for the sake of readability.

Reply:
> So, I have converted the COLORREF color & bool hilite assignments to direct brace initialization. Since nppParams is taking reference assignment, my old habit still favors copy initialization for it.

I have no problem with such way for initialization. Though I would use the traditional way - You know that you can't teach an old dog new tricks :)